### DISCUSSION TITLE
Firefox addon " I don't care about cookies" is pure cancer.

### DISCUSSION THREAD

**Discussion by eskiq** [gvolpe/nix-config#257] [General]
https://addons.mozilla.org/en-US/firefox/addon/i-dont-care-about-cookies/
More about it in reviews.
There is alternative called [I still don't care about cookies](https://addons.mozilla.org/en-US/firefox/addon/istilldontcareaboutcookies/).

**Comment by gvolpe** [comment]
Thank you for bringing this to my attention, I was not aware. Addressed in https://github.com/gvolpe/nix-config/commit/f1afbacea138fed67db1d90a36200991acb7e451
